### PR TITLE
Move from simple list to numbered list

### PR DESCRIPTION
**Pull request by DelazJ**
Because those are kind of steps to follow instead of simple enumeration, it looks better to show them as such.

### CODE REVIEW

File: source/docs/user_manual/managing_data_source/opening_data.rst [hunk 1]
```diff
@@ -468,16 +468,16 @@ database the frame in the lower half of the dialog is populated with the list of
...
+#. At the top, set a :guilabel:`Group name` to group the drawing files in the
+   project.
+#. Check layers to show: Each selected layer is added to an ad hoc group which
+   contains vector layers for the point, line, label and area features of the
+   drawing layer. The style of each layer is setup so that it resembles the look
+   it originally had in \*CAD.
+#. Check whether layer should be visible at opening.
+#. Alternatively using the |checkbox| :guilabel:`Merge layers` option places all
+   layers in a single group.
+#. Press **[OK]** to open the layers in QGIS.
```
Comment (ghtmtt) [inline source/docs/user_manual/managing_data_source/opening_data.rst]: I guess **[OK]** and other button occurrences in this PR have been replaced with PR #3123

File: source/docs/user_manual/managing_data_source/supported_data.rst [hunk 2]
```diff
@@ -96,11 +96,11 @@ Spatial indexes used by QGIS have a :file:`.qix` extension.
...
 
-*  Load a shapefile (see :ref:`browser_panel`);
-*  Open the :guilabel:`Layer Properties` dialog by double-clicking on the
+#. Load a shapefile (see :ref:`browser_panel`).
+#. Open the :guilabel:`Layer Properties` dialog by double-clicking on the
    shapefile name in the legend or by right-clicking and choosing
-   :menuselection:`Properties` from the context menu.
-*  In the :guilabel:`Source` tab, click the **[Create Spatial Index]** button.
+   :menuselection:`Properties...` from the context menu.
+#. In the :guilabel:`Source` tab, click the **[Create Spatial Index]** button.
```
Comment (ghtmtt) [inline source/docs/user_manual/managing_data_source/supported_data.rst]: same of above

File: source/docs/user_manual/working_with_vector/vector_properties.rst [hunk 3]
```diff
@@ -513,13 +513,14 @@ other is represented by a size.
...
-* Then, apply a proportional symbology on the classes: click on the symbol
-  **[Change]** button above the classification frame: you get the
-  :ref:`symbol-selector` dialog from which, as seen above, you can rescale the
-  size or width using the |dataDefined| :ref:`data defined override
-  <data_defined>` widget.
+#. First apply a categorized or graduated rendering on a layer, using the same
+   type of symbol for all the classes.
+#. Then, apply a proportional symbology on the classes:
+
+   #. Click on the **[Change]** button above the classification frame: you get the
```
Comment (ghtmtt) [inline source/docs/user_manual/working_with_vector/vector_properties.rst]: same of above

File: source/docs/user_manual/working_with_vector/vector_properties.rst [hunk 4]
```diff
@@ -554,23 +555,23 @@ if you want to enable symbol levels or use only the first-matched rule.
...
+   |projectProperties| :sup:`Edit rule` or |signPlus| :sup:`Add rule` button.
+#. In the :guilabel:`Edit Rule` dialog that opens, you can define a label
+   to help you identify each rule. This is the label that will be displayed
+   in the :guilabel:`Layers Panel` and also in the print composer legend.
+#. Manually enter an expression in the text box next to the |radioButtonOn|
+   :guilabel:`Filter` option or press the |expression| button next to it to open
+   the expression string builder dialog.
+#. Use the provided functions and the layer attributes to build an :ref:`expression
+   <vector_expressions>` to filter the features you'd like to retrieve. Press
+   the **[Test]** button to check the result of the query.
```
Comment (ghtmtt) [inline source/docs/user_manual/working_with_vector/vector_properties.rst]: same

File: source/docs/user_manual/working_with_vector/vector_properties.rst [hunk 5]
```diff
@@ -554,23 +555,23 @@ if you want to enable symbol levels or use only the first-matched rule.
...
+#. You can enter a longer label to complete the rule description.
+#. You can use the |checkbox| :guilabel:`Scale Range` option to set scales at which
+   the rule should be applied.
+#. Finally, configure the :ref:`symbol to use <symbol-selector>` for these features.
+#. And press **[OK]**.
```
Comment (ghtmtt) [inline source/docs/user_manual/working_with_vector/vector_properties.rst]: again same

**Comment by DelazJ** [comment]
Yes @ghtmtt they are fixed there ( ie welcome conflicts!)

**Pull request merged**